distribution/ddf-common/src/main/resources/etc/certs/CertNew.sh

```diff
@@ -37,13 +37,14 @@ fi
 if [[ $1 && $2 ]]; then
     PARAM1="$1"
     PARAM2="$2"
+    shift
```

Comment:
What is the problem with keeping PARAM1 and PARAM2, they represent respectively the first and second parameters retrieved from the parameters?